rpc_proxy.go

```diff
@@ -165,6 +166,9 @@ type rpcProxy struct {
 
 	grpcServer   *grpc.Server
 	grpcWebProxy *grpcweb.WrappedGrpcServer
+
+	started bool
```

Comment:
Could use an atomic bool alternatively.

Reply:
Still getting thru the diff, but could also use an event type to allow callers to get notifications for when things are fully up vs polling.